configuration/configuration.go

```diff
@@ -86,6 +86,12 @@ type Configuration struct {
 			ClientCAs []string `yaml:"clientcas,omitempty"`
 		} `yaml:"tls,omitempty"`
 
+		// Headers is a set of headers to include in HTTP responses. A common
+		// use case for this would be security headers such as
+		// Strict-Transport-Security. The map keys are the header names, and
+		// the values are the associated header payloads.
+		Headers map[string]string `yaml:"headers,omitempty"`
```

Comment:
`http.Header` is `map[string][]string`, which I don't think we want here. We want keys to be header names and values to be payloads. If a header should appear multiple times, the user can list the header multiple times in the configuration file.